tensorflow/core/framework/register_types.h

```diff
@@ -58,7 +58,11 @@ limitations under the License.
 // Note: macros are defined in same order as types in types.proto, for
 // readability.
 #define TF_CALL_float(m) m(float)
+#if defined(_WINDOWS) && !defined(GOOGLE_CUDA) && !defined(__AVX__) && !defined(__AVX2__) // Not supported on Eigen for SSE
```

Comment:
I'm a little concerned about this line. How many things failed without it? If the number of kernels to disable is not overwhelming, we might be able to do this at the CMake or kernel registration level... disabling an entire type seems drastic!

Would the predicate here be equivalent to `#if defined(PLATFORM_WINDOWS) && defined(DEBUG)`? If so, the intent might be clearer.